ui/src/testing/default-server-config.ts

```diff
@@ -77,6 +77,7 @@ const defaultServerConfig: ConfigResponse = {
   tanagraBaseUrl: 'https://test.fake-research-aou.org',
   enableGKEAppPausing: false,
   enableGKEAppMachineTypeChoice: false,
+  enablePublishedWorkspacesViaDb: true,
```

Comment:
I prefer false here for feature flags, because it makes us think of Prod as the default case, and Test as something temporary.

This may require adding some tests for the currently-untested case.

Reply:
Fair point. I swapped that and added a couple tests for when the feature flag is false.